src/components/shared/map/layers/styles/gps-image-popup.scss

```diff
@@ -0,0 +1,16 @@
+/* Image on popup */
+.ol-popup img {
+  max-width: 100px;
```

Comment:
We should probably use em here and not px so it scales correctly on mobile devices. :) I think there are other places where we also use px. We should discuss this in the planning.

Reply:
Good point, changed. And yes, let's discuss this, the whole frontend needs some love... soon :D